import-service/src/functions/importProductsFile/handler.ts

```diff
@@ -0,0 +1,35 @@
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import * as AWS from 'aws-sdk';
+
+export const importProductsFile = async (event) => {
+  console.log('[importProductsFile] called, arguments: ', JSON.stringify(event.queryStringParameters));
+
+  const queryParams = event.queryStringParameters;
+
+  if (!queryParams?.name) {
+    return formatJSONResponse('Not found "name" query parameter', 400);
+  }
+
+  const { name } = queryParams;
```

Comment:
1. `name` property can be destructured from the event.queryStringParameters object once at the top of the function
2. s3 can be initialized at the top of the file, or in a separate file, and then imported
3. bucket name should be moved to the .env file as an environment variable

Reply:
Done.